diff --git a/node_modules/react-native-permissions/ios/RNPermissions.m b/node_modules/react-native-permissions/ios/RNPermissions.m
index 96d20b4..066c29b 100644
--- a/node_modules/react-native-permissions/ios/RNPermissions.m
+++ b/node_modules/react-native-permissions/ios/RNPermissions.m
@@ -184,16 +184,16 @@ - (NSDictionary *)constantsToExport {
 #endif
 
 #if RCT_DEV
-  if ([available count] == 0) {
-    NSMutableString *message = [NSMutableString new];
+  // if ([available count] == 0) {
+  //   NSMutableString *message = [NSMutableString new];
 
-    [message appendString:@"⚠  No permission handler detected.\n\n"];
-    [message appendString:@"• Check that you link at least one permission handler in your Podfile.\n"];
-    [message appendString:@"• Uninstall this app, delete your Xcode DerivedData folder and rebuild it.\n"];
-    [message appendString:@"• If you use `use_frameworks!`, follow the workaround guide in the project README."];
+  //   [message appendString:@"⚠  No permission handler detected.\n\n"];
+  //   [message appendString:@"• Check that you link at least one permission handler in your Podfile.\n"];
+  //   [message appendString:@"• Uninstall this app, delete your Xcode DerivedData folder and rebuild it.\n"];
+  //   [message appendString:@"• If you use `use_frameworks!`, follow the workaround guide in the project README."];
 
-    RCTLogError(@"%@", message);
-  }
+  //   RCTLogError(@"%@", message);
+  // }
 #endif
 
   return @{ @"available": available };
